GliaWidgets/SecureConversations/SecureConversations.Availability.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+
+extension SecureConversations {
+    struct Availability {
+        typealias CompletionResult = (Result<Bool, CoreSdkClient.SalemoveError>) -> Void
+        var environment: Environment
+
+        func checkSecureConversationsAvailability(completion: @escaping CompletionResult) {
+            environment.listQueues { queues, error in
+                if let error = error {
+                    completion(.failure(error))
+                    return
+                }
+
+                self.checkQueues(queues, completion: completion)
+            }
+        }
+
+        private func checkQueues(
+            _ queues: [CoreSdkClient.Queue]?,
+            completion: (Result<Bool, CoreSdkClient.SalemoveError>) -> Void
+        ) {
+            guard let queues = queues, isSecureConversationsAvailable(in: queues) else {
+                completion(.success(false))
+
```

Comment:
would you like to keep this newline for some reason or we can remove it?

Reply:
Removed fbb0d8ef7636a0daeeaf95d166b4084fca2f42f9